plugins/jobs/server/job_rest.py

```diff
@@ -130,3 +141,19 @@ def updateJob(self, job, log, overwrite, notify, status, progressTotal, progress
     )
     def deleteJob(self, job, params):
         self.model('job', 'jobs').remove(job)
+
+    @access.admin
+    @autoDescribeRoute(
+        Description('Get metadata of all jobs')
+        .errorResponse('Admin access was denied for the job.', 403)
+    )
+    def allJobsMeta(self, params):
+        return self.model('job', 'jobs').getAllTypesAndStatuses(user='all')
+
+    @access.user
+    @autoDescribeRoute(
+        Description('Get metadata of jobs of current user')
+    )
+    def jobsMeta(self, params):
```

Comment:
Here too, `jobsTypesAndStatuses`.  Or maybe there is a word that means 'types and statuses' but is different from 'metadata', good luck if you want to find it.